### DISCUSSION TITLE
بات تلگرام

### DISCUSSION THREAD

**Discussion by roseshayan** [Gozargah/Marzban#399] [Q&A]
درود وقتتون بخیر داخل تنظیمات .env یه بخش هست که برای بات تلگرامه و ما باید توکن ربات و آیدی عددی ادمین رو وارد کنیم.
من وقتی وارد میکنم و از حالت کامنت درمیارم مرزبان رو که ریستارت میکنم برام مرزبان ران نمیشه! کسی میدونه چرا؟
آموزشی در این زمینه هست؟

**Comment by NarimanJelodar** [comment]
اروری که میگیری چیه اسکرین بزار شاید بتونم کمکت کنم
